README.md

```diff
@@ -71,7 +53,7 @@ sim.client("client", async move {
 sim.run();
 ```
 
-For more examples, check out the [tests](tests) directory.
+For more examples (with simulated networking), check out the [tests](tests) directory.
```

Comment:
Can you link directly to an example, such as the echo example?

Reply:
yup